### PR TITLE
 refactor(ivy): cleanup tests from render3

### PR DESCRIPTION
**Pull request by cexbrayat**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

`render3` contains tests for JS blocks that are no longer planned to be supported.

## What is the new behavior?

The `di_spec.ts` tests were already migrated to acceptance with `ngFor`/`ngIf`, but were leftover in case JS blocks ended up supported in Ivy. They are now removed.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/core/test/render3/control_flow_spec.ts [hunk 1]
```diff
@@ -1,950 +0,0 @@
...
- * found in the LICENSE file at https://angular.io/license
- */
-
-import {ɵɵdefineComponent} from '../../src/render3/definition';
-import {ɵɵcontainer, ɵɵcontainerRefreshEnd, ɵɵcontainerRefreshStart, ɵɵelement, ɵɵelementEnd, ɵɵelementStart, ɵɵembeddedViewEnd, ɵɵembeddedViewStart, ɵɵselect, ɵɵtext, ɵɵtextBinding} from '../../src/render3/instructions/all';
-import {RenderFlags} from '../../src/render3/interfaces/definition';
-import {ComponentFixture, TemplateFixture, createComponent} from './render_util';
-
-describe('JS control flow', () => {
-  it('should work with if block', () => {
```
Comment (kara) [inline packages/core/test/render3/control_flow_spec.ts]: Do we have an existing test with *ngIf or *ngFor for all of these? Want to check that we don't lose coverage before deleting.

**Comment by AndrewKushnir** [comment]
@cexbrayat @kara does it make sense to remove the JS blocks tests alongside with the code (core, compiler) that supports JS blocks?

**Comment by cexbrayat** [comment]
@kara I removed the second commit and rebased. The PR now only contains deletions of tests in `di_spec.ts`, that I already migrated to `acceptance` with `*ngIf/*ngFor`.

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
merge-assistance: Saucelabs failure was due to master being red

**Pull request closed without merging**